README.md

```diff
@@ -12,11 +12,11 @@ TinyGo is a project to bring Go to microcontrollers and small systems with a
 single processor core. It is similar to [emgo](https://github.com/ziutek/emgo)
 but a major difference is that I want to keep the Go memory model (which implies
 garbage collection of some sort). Another difference is that TinyGo uses LLVM
-internally instead of emitting C, which hopefully leads to smaller and more
-efficient code and certainly leads to more flexibility.
+internally instead of emitting C, which hopefully leads to small,
+efficient, and flexible code.
```

Comment:
Using LLVM doesn't lead to more flexible code. It leads to a more flexible compiler design which in turn makes it easier to optimize the generated code (in fact, I'm already using LLVM optimizatons mixed with custom optimizations producing a better result overall).

Perhaps the sentence would be correct with something like this:

> Another difference is that TinyGo uses LLVM internally instead of emitting C, which is a more flexible compiler design, making it easier to emit small and efficient code.